Packages.props

```diff
@@ -18,9 +18,9 @@
 		<PackageReference Update="Microsoft.Azure.Management.Sql" Version="1.41.0-preview" />
 
 		<PackageReference Update="Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider" Version="1.1.1" />
-		<PackageReference Update="Microsoft.Data.SqlClient" Version="2.1.3"/>
+		<PackageReference Update="Microsoft.Data.SqlClient" Version="3.0.0"/>
```

Comment:
Do you know what changes there were between v2 and v3? Major version bumps like this should be taken with much more caution so it would be best to know what differences to expect and whether they'll affect us.

Have you done any testing of connections/queries on the various platforms as well?

Reply:
Yes, I have done connections/querying along with testing on Windows and I'll take Kim/Leila's help for Mac testing.

I see these 3 are the [breaking changes](https://github.com/dotnet/SqlClient/blob/main/release-notes/3.0/3.0.0.md) with MDS3, #1 and #2 looks ok to me, for the #3 one I debugged the sqlDataReader code lines just to observe the functionality and seems good too.
1. The minimum supported .NET Framework version has been increased to v4.6.1. .NET Framework v4.6.0 is no longer supported.
2. User Id connection property now requires Client Id instead of Object Id for User-Assigned Managed Identity 
3. SqlDataReader now returns a DBNull value instead of an empty byte[]. Legacy behavior can be enabled by setting AppContext switch Switch.Microsoft.Data.SqlClient.LegacyRowVersionNullBehavior

Reply:
# sign has a special meaning, it will automatically be linked to PR or issues in this repo. you might want to edit the description so that people won't get confused :)

Reply:
![image](https://user-images.githubusercontent.com/13777222/139954342-2a52746e-d0e9-49eb-a365-690e75148e4d.png)

Reply:
I've updated your comments and removed the links

Reply:
Thanks Alan!